apps/dashboard/src/components/welcome/inbox-framework-guide.tsx

```diff
@@ -123,36 +169,103 @@ export function InboxFrameworkGuide({
+                whileHover="hover"
+                animate={framework.name === selectedFramework.name ? 'hover' : 'initial'}
+                className="relative"
+              >
+                <Card
+                  onClick={() => handleFrameworkSelect(framework)}
+                  className={`flex h-[100px] w-[100px] flex-col items-center justify-center border-none p-6 shadow-none hover:cursor-pointer ${
+                    framework.name === selectedFramework.name ? 'bg-neutral-100' : ''
+                  }`}
+                >
+                  <CardContent className="flex flex-col items-center gap-3 p-0">
+                    <motion.div variants={iconVariants} className="relative text-2xl">
+                      {framework.icon}
+                    </motion.div>
+                    <span className="text-sm text-[#525866]">{framework.name}</span>
+                  </CardContent>
+                </Card>
+              </motion.div>
+            ))}
+          </div>
+        </div>
+
+        {!['Remix', 'Native', 'Angular', 'JavaScript'].includes(selectedFramework.name) ? (
+          <div className="flex flex-col gap-4">
+            <h3 className="text-sm font-medium text-[#525866]">Choose installation method</h3>
+            <Tabs
+              defaultValue="cli"
+              value={installationMethod}
+              onValueChange={(value) => setInstallationMethod(value as 'cli' | 'manual')}
             >
-              <CardContent className="flex flex-col items-center gap-3 p-0">
-                <motion.div variants={iconVariants} className="relative text-2xl">
-                  {framework.icon}
-                </motion.div>
-                <span className="text-sm text-[#525866]">{framework.name}</span>
-              </CardContent>
-            </Card>
-          </motion.div>
-        ))}
+              <TabsList className="inline-flex items-center gap-2 bg-transparent p-0">
+                <TabsTrigger
+                  value="cli"
+                  className="relative text-xs font-medium text-[#525866] transition-colors hover:text-[#dd2476] data-[state=active]:text-[#dd2476]"
+                >
+                  CLI
+                </TabsTrigger>
+                <TabsTrigger
+                  value="manual"
+                  className="relative text-xs font-medium text-[#525866] transition-colors hover:text-[#dd2476] data-[state=active]:text-[#dd2476]"
+                >
+                  Manual
+                </TabsTrigger>
+              </TabsList>
+              <div className="relative mt-2 overflow-hidden pl-0">
+                <AnimatePresence mode="wait">
+                  {installationMethod === 'cli' && (
+                    <motion.div
+                      key="cli"
+                      initial={{ opacity: 0 }}
+                      animate={{ opacity: 1 }}
+                      exit={{ opacity: 0 }}
+                      transition={{ duration: 0.15 }}
```

Comment:
```suggestion
                      {...fadeIn}
```


please use the `fadeIn` constant from the `animations.ts` file, also please update it in the other places

Reply:
I've addressed it.